felix/config/param_types.go

```diff
@@ -307,6 +307,19 @@ func (p *Ipv4Param) Parse(raw string) (result interface{}, err error) {
 	return
 }
 
+type Ipv6Param struct {
+	Metadata
+}
+
+func (p *Ipv6Param) Parse(raw string) (result interface{}, err error) {
+	res := net.ParseIP(raw)
```

Comment:
This should really check that it's actually an IPv6 address (and Ipv4Param should check that it's really IPv4!)  Please can you add that.